blocks/footer/footer.js

```diff
@@ -92,8 +92,15 @@ export default async function decorate(block) {
       sources.classList.add('open');
     });
 
-    const buttonSibling = block.querySelector('.footer-content p.button-container');
-    buttonSibling.parentElement.insertBefore(viewSourceBtn, buttonSibling.nextElementSibling);
+    const footerButtonsWrapperDiv = createTag('div');
+    const contactUsWrapperPara = createTag('p', { class: 'button-container' });
+    const footerContentFlexDiv = block.querySelector('.footer-content div');
+    const contactUsAnchor = block.querySelector('.footer-content .button-container a');
+    contactUsAnchor.classList.add('button', 'primary');
+    footerContentFlexDiv.appendChild(footerButtonsWrapperDiv);
+    contactUsWrapperPara.appendChild(contactUsAnchor);
+    footerButtonsWrapperDiv.appendChild(contactUsWrapperPara);
+    footerButtonsWrapperDiv.appendChild(viewSourceBtn);
```

Comment:
To reduce DOM accesses, you can append both children at once:
```suggestion
    footerButtonsWrapperDiv.appendChild(contactUsWrapperPara, viewSourceBtn);
```

Reply:
I'm surprised that to remove the "Designed By" logo.  I assume this is actually to remove the logo and insert the 2 buttons right?

Reply:
Per [specification](https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild#example), there is no append multiple children support. But I simplified by chaining statements. also since we extract `footerButtonsWrapperDiv`, we dont access DOM second time. 

Done resolved this comment.